File: O_Maior.py
a, b, c = input().split()
a = int(a)
b = int(b)
c = int(c)
maiorab = (a + b + abs(a - b)) / 2
maiorac = int((maiorab + c + abs(maiorab - c))/2)
print("{} eh o maior".format(maiorac))

File: O_Maior.js
const fs = require("fs");
const r = fs.readFileSync(0, "utf8");
let line = r.split(" ");
let a = parseFloat(line[0]);
let b = parseFloat(line[1]);
let c = parseFloat(line[2]);
let maior = (a+b+Math.abs(a-b))/2;
maior = (maior+c+Math.abs(maior-c))/2;
console.log(`${maior} eh o maior`);